README.md

```diff
@@ -215,27 +215,16 @@ be added as follows:
 ```
 > rustup target add wasm32-unknown-unknown
 > cargo install -f wasm-bindgen-cli
+> deno install --unstable -A -f -n wasmbuild https://raw.githubusercontent.com/denoland/wasmbuild/d51c673a321ecf4f336a467bb1a4a5327e783481/main.ts
 ```
 
 > ⚠️ Note that the `wasm-bindgen-cli` should match the version of `wasm-bindgen`
 > in this crate and be explicitly set using the `--version` flag on install.
 
-Also, the build script (`_build.ts`) requires the Deno CLI to be installed and
-available in the path. If it is, the script should _just work_:
-
-```
-> ./_build.ts
-```
-
-But can be manually invoked like:
-
 ```
-> deno run --unstable _build.ts
+> wasmbuild --no-default-features --features wasm
```

Comment:
Until wasmbuild stabilizes we should probably continue using a script with a pegged version (ex. getting https://github.com/denoland/wasmbuild/issues/3 done)